src/styles/typography.ts

```diff
@@ -41,5 +46,10 @@ export const typography = {
     fontFamily: fontByWeight.medium,
     marginBottom: 14 * 0.35
   },
+  body2: {
```

Comment:
`body2` seems kinda ambiguous to me. I guess it does make sense following h1, h2, etc. pattern though

Reply:
yeah its... not the best, im kinda throwing out values here to try and categorize stuff, just so we can take stock of which values we are centering on, and then we can bring to julian to nail down naming convention?